.github/workflows/gh_pages.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy documentation to GH Pages
+
+on:
+  pull_request:
+  merge_group:
```

Comment:
Let's only do this on main and on dispatch.

Reply:
Makes sense, thanks!